src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/wwwroot/js/site.js

```diff
@@ -0,0 +1,6 @@
+﻿"use strict";
+
+// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+// for details on configuring this project to bundle and minify static web assets.
+
+// Write your Javascript code.
```

Comment:
Wrong capitalization of `JavaScript`. Is this wrong in other templates?

Reply:
It is indeed wrong in *some* other templates. I filed https://github.com/aspnet/AspNetCore/issues/10414 to fix other templates. But please fix this one here.

Reply:
J A V A S C R I P T

Reply:
jAvAsCrIpT

Reply:
Sounds too close to Java... Clearly this should be ECMAScript 😆